File: leetcode/3656-minimum-number-of-operations-to-make-elements-in-array-distinct/solution.c
int minimumOperations(int* nums, int n) {
    int vis[101] = {};
    int i = n-1;
    for(;i>=0 && !vis[nums[i]];i--) {
        vis[nums[i]] = true;
    }
    return (i+3)/3;
}

File: leetcode/3656-minimum-number-of-operations-to-make-elements-in-array-distinct/solution.rs
impl Solution {
    pub fn minimum_operations(nums: Vec<i32>) -> i32 {
        let mut vis = vec![false;101];
        let mut i = nums.len();
        for &x in nums.iter().rev() {
            if vis[x as usize] {
                break;
            }
            vis[x as usize] = true;
            i -= 1;
        }
        return (i as i32 + 2)/3;
    }
}
